src/Sylius/Bundle/CoreBundle/Fixture/OptionsResolver/LazyOption.php

```diff
@@ -133,4 +135,8 @@ public static function findOneBy(RepositoryInterface $repository, string $field)
             return $repository->findOneBy([$field => $previousValue]);
         };
     }
+
+    private static function findLimitedAll(RepositoryInterface $repository, int $limit = 2000) {
```

Comment:
Move bracket to the next line and add a return type

Reply:
Fixed